### ISSUE TITLE
Support for column renaming

### ISSUE THREAD

**Issue by gVirtu** [PGHist/PGHist#4]
Hello, thank you for your work on this tool, I've been taking a look at it over the past few days and really enjoyed it so far! :)

One particular point of concern is that renaming a column on a hist_enabled table currently seems to be equivalent to [dropping and then adding a column](https://github.com/PGSuite/PGHist/blob/main/pghist_init.sql#L424-L452) to the `hist_data$table`. This results in wiping the history of the column data.

Can we do anything to improve handling of this particular scenario?

Even if not possible to automate it, maybe making it so that this kind of `rename` operation raises an exception unless the respective target columns in `hist_data$table` already exist with the new name (i.e: were manually renamed first) would be preferred to silently dropping the column history.

Would love to hear your thoughts on this.

Cheers, and thanks once more.

**Comment by PGSuite** [comment]
Hi, thanks for the feedback.

Yes, renaming a column deletes the history.
I will think about it.

**Comment by PGSuite** [comment]
Hi

Fixed.
If interested, see function pghist.hist_event_column_rename

Regards!

**Comment by jhonathas** [comment]
@PGSuite 

When testing the field rename, I noticed that an error occurs when executing the `pghist.hist_event_ddl_command()` function because the `tmp_log_insert` function is executed but does not exist. Since the line that calls this function has a -- *** https://github.com/PGSuite/PGHist/blob/main/pghist_init.sql#L450 . I was wondering if it wasn't some code that should have been commented out or deleted since this function is not used anywhere else.

I removed this line and am using the script, apparently without error. I can rename the fields without losing data and the history works.
